LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-Copyright (c) 2018 Hydrogen Epoch of Reionization Array (HERA)
+Copyright (c) 2019 Hydrogen Epoch of Reionization Array (HERA)
```

Comment:
I updated the copyright dates while I was waiting for tests to run. Sorry reviewers.